pkg/kubelet/dockershim/libdocker/kube_docker_client.go

```diff
@@ -123,6 +123,19 @@ func (d *kubeDockerClient) InspectContainer(id string) (*dockertypes.ContainerJS
 	return &containerJSON, nil
 }
 
+func (d *kubeDockerClient) InspectContainerWithSize(id string) (*dockertypes.ContainerJSON, error) {
+	ctx, cancel := d.getTimeoutContext()
+	defer cancel()
+	containerJSON, _, err := d.client.ContainerInspectWithRaw(ctx, id, true)
```

Comment:
Add comments explaining what this call does -- `Return the size of container as fields SizeRw and SizeRootFs.`

Reply:
Fixed, thanks for suggestion.